catroid/src/main/res/values/strings.xml

```diff
@@ -1366,6 +1367,7 @@
     <!-- Raspi Settings -->
     <string name="raspi_settings_title_name">Raspberry Pi settings</string>
     <string name="raspi_settings_gpio_version">GPIO Version</string>
+    <string name="error_connecting_to">Error: Could not connect to %s : %d%%</string>
```

Comment:
Ahm, why the two `%` at the end?

Reply:
i was getting a lint error without %% .
but now it's fine -_- i don't know how or why! just a magic happened :)))